src/libasr/string_utils.cpp

```diff
@@ -116,7 +116,7 @@ std::string read_file(const std::string &filename)
     std::vector<char> bytes(filesize);
     ifs.read(&bytes[0], filesize);
 
-    return std::string(&bytes[0], filesize);
+    return replace(std::string(&bytes[0], filesize), "\r\n", "\n");
```

Comment:
Let's open an issue that our tokenizer/parser should be updated/fixed to produce the same tokens/parse-tree on windows and unix.